src/TodoList.js

```diff
@@ -17,12 +17,13 @@ const todoList = [
     }
   ];
   
-const TodoList = () => {
+const TodoList = ({todoList}) => {
+  const {id, title} = todoList;
     return (
         <ul>
             {todoList.map(function(item) {
                 return (
-                    <TodoListItem key={item.id} item={item} />
+                    <TodoListItem key={id} item={item} />
```

Comment:
then this `key` prop will be assigned properly to the item's `id`, and not `null` as it is currently

Reply:
done